Libraries/LogHelpers.js

```diff
@@ -30,6 +30,28 @@ function get_version_and_board(log) {
 
     if ('MSG' in log.messageTypes) {
         const MSG = log.get("MSG")
+
+        const build_types = {
```

Comment:
Does this need to be in the `if ('MSG' in log.messageTypes) {` bit? I don't think it uses anything from `MSG`, but it does use `VER`, I think it should be in the above `if`.